parachain/runtime/src/lib.rs

```diff
@@ -352,15 +358,19 @@ parameter_types! {
     pub const LaunchPeriod: BlockNumber = 2 * DAYS;
     pub const VotingPeriod: BlockNumber = 2 * DAYS;
     pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
-    pub MinimumDeposit: Balance = 1000; // TODO: adjust
+    pub const MinimumDeposit: Balance = 100 * CENTS;
```

Comment:
What is `CENTS`? I guess we want to measure minimum deposit in `KINT`/`INTR` where smallest denomination would be in `planck`